external/upstream/boost/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+find_package(Boost ${Boost_MINIMUM_REQUIRED} QUIET COMPONENTS "${BOOST_COMPONENTS_REQUIRED}")
```

Comment:
I could be wrong, but I'm guessing you copy/pasted this from somewhere. If that's the case you need to attribute the source. That said as my previous comment suggested you don't want to use Boost in this project.